dev-lang/ispc/ispc-9999.ebuild

```diff
@@ -35,9 +35,9 @@ DEPEND="
 	"
 
 PATCHES=(
-	"${FILESDIR}/${PN}-1.13.0-cmake-gentoo-release.patch"
-	"${FILESDIR}/${PN}-1.13.0-llvm-10.patch"
-	"${FILESDIR}/${PN}-1.13.0-werror.patch"
+	"${FILESDIR}/${PN}-1.14.0-cmake-gentoo-release.patch"
+	"${FILESDIR}/${PN}-1.14.0-llvm-10.patch"
+	"${FILESDIR}/${PN}-1.14.0-werror.patch"
 )
 
 src_configure() {
```

Comment:
The $(usex arm) thing is odd, but not wrong. I wonder if we need it on arm64 too? Or any other architectures (different options)?

Reply:
Hmm, I don't have any arm computers to check this with...

Reply:
Well, this isn't even keyworded on arm or arm64, so no need to worry too much. We could look at the build system to see what it's doing for fun though.